src/command_modules/azure-cli-eventgrid/azure/cli/command_modules/eventgrid/_params.py

```diff
@@ -27,15 +29,33 @@
     nargs='+'
 )
 
+odata_query_type = CLIArgumentType(
+    help="The OData query used for filtering the list results. Filtering is currently allowed on the Name property only. The supported operations include: CONTAINS, eq (for equal), ne (for not equal), AND, OR and NOT.",
+    options_list=['--odata-query']
+)
+
+
+domain_name_type = CLIArgumentType(
+    help='Name of the domain.',
+    arg_type=name_type,
+    options_list=['--domain-name'],
+    completer=get_resource_name_completion_list('Microsoft.EventGrid/domains'))
 
-def load_arguments(self, _):
+domain_topic_name_type = CLIArgumentType(
+    help='Name of the domain topic.',
+    arg_type=name_type,
+    options_list=['--domain-topic-name'],
+    completer=get_resource_name_completion_list('Microsoft.EventGrid/domains/topic'))
+
+
+def load_arguments(self, _):    # pylint: disable=too-many-statements
     with self.argument_context('eventgrid') as c:
         c.argument('resource_group_name', arg_type=resource_group_name_type)
         c.argument('location', arg_type=get_location_type(self.cli_ctx))
         c.argument('tags', arg_type=tags_type)
         c.argument('included_event_types', arg_type=included_event_types_type)
         c.argument('labels', arg_type=labels_type)
-        c.argument('endpoint_type', arg_type=get_enum_type(['webhook', 'eventhub', 'storagequeue', 'hybridconnection'], default='webhook'))
+        c.argument('endpoint_type', arg_type=get_enum_type(['webhook', 'eventhub', 'storagequeue', 'hybridconnection', 'servicebusqueue'], default='webhook'))
```

Comment:
Are these still not enums in your SDK??

Reply:
yes. Same behavior as previous GA and extension.

Reply:
You are right about the enum value in EventGrid Swagger. But your initial question was about the generated Python SDK which lacks this value and hence we don't use it here. We started a thread with Python SDK team who owns the generation of Python SDK to determine if there is a bug in the autorest. Added you to the thread for your awareness but this should be addressed separately so we don't block this PR for the GA version.